src/getMigrateSavedWidgets.ts

```diff
@@ -68,7 +68,7 @@ export const getMigrateSavedWidgets =
 
     for (const fileId in content.children) {
       if (errorReport.widgets?.[fileId] && behaviorOnError !== "keep-going") {
-        return;
+        continue;
```

Comment:
Previously, returning here would cause the entire script to "short-circuit" causing none of the remaining widgets to be migrated.